packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/scopes/index.md

```diff
@@ -26,10 +26,12 @@ The following table shows the scopes that are currently available as a part of E
 | `okta.inlineHooks.manage`| Allows the app to create and manage Inline Hooks in your Okta organization.| [Inline Hooks API](/docs/reference/api/inline-hooks/)|
 | `okta.inlineHooks.read` | Allows the app to read information about Inline Hooks in your Okta organization.| [Inline Hooks API](/docs/reference/api/inline-hooks/)|
 | `okta.logs.read`         | Allows the app to read information about System Log entries in your Okta organization| [System Log API](/docs/reference/api/system-log/)|
+| `okta.policies.manage`   | Allows the app to manage policies in your Okta organization             | [Policy API](/docs/reference/api/policy/) |
```

Comment:
I believe I have updated the developer doc regarding okta.policies.manage and okta.policies.read in another PR https://github.com/okta/okta-developer-docs/pull/916. Something I missed here?

Reply:
Ooops.. I just hadn't pulled recently enough! I'll remove these lines 😄